docs/CONTRIBUTING.md

```diff
@@ -52,7 +52,7 @@ application icon if built on Linux or Windows due to non-trivial icon generation
 on these platforms. To develop the source code you will need the following
 setup:
 
-- **The Rust Toolchain (nightly-2019-11-04)**
+- **The Rust Toolchain (nightly-2021-05-12)**
```

Comment:
This also needs updating in the commands in the section below. (GH won't let me comment in the line directly as it has not been changed).
`rustup toolchain install nightly-2019-11-04`
to 
`rustup toolchain install nightly-2021-05-12`

Reply:
Is fixed